### ISSUE TITLE
Keyboard navigation playback in Table does not work

### ISSUE THREAD

**Issue by vaadin-bot** [vaadin/testbench#274]
**Originally by @Artur-**

___
Recording keyboard navigation events in Table works but on playback nothing happens. See attached test case for demo.vaadin.com/sampler
